src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.js

```diff
@@ -319,4 +320,33 @@ export default class GeBatchGiftEntryTable extends LightningElement {
         }));
     }
 
+    handleMenuItemSelect(event) {
+        if (event.detail.value === 'selectcolumns') {
+            const selectColumns = new CustomEvent('selectcolumns', {
+                detail: {
+                    options: this.getAllColumns()
+                        .map(({label, fieldName}) => ({
+                            label, value: fieldName
+                        })),
+                    values: this.computedColumns
+                        .map(({fieldName}) => fieldName)
+                }
+            });
+            this.dispatchEvent(selectColumns);
+        }
+    }
+
+    get tableMenuQaLocator() {
+        return 'button Show menu';
+    }
+
+    get selectBatchTableColumnsQaLocator() {
+        return `link ${this.CUSTOM_LABELS.geSelectBatchTableColumns}`;
+
+    }
+
+    get selectBatchTableColumns() {
+        return this.CUSTOM_LABELS.geSelectBatchTableColumns;
```

Comment:
Any reason we shouldn't reference `CUSTOM_LABELS.geSelectBatchTableColumns` directly in the markup?

Reply:
No, I think that would be fine here!